midenc-compile/src/compiler.rs

```diff
@@ -201,6 +201,29 @@ pub struct Compiler {
         )
     )]
     pub link_libraries: Vec<LinkLibrary>,
+    /// Specify path prefixes to try when resolving relative paths from DWARF debug info
+    #[cfg_attr(
+        feature = "std",
+        arg(
+            long = "trim-path-prefix",
+            value_name = "PATH",
+            help_heading = "Compiler"
+        )
+    )]
+    pub trim_path_prefixes: Vec<PathBuf>,
```

Comment:
Let's move this to `UnstableOptions` as well, since this is also an internal implementation detail that is pretty specific to the Rust compilation pipeline and may change at any time depending on how Rust/LLVM decides to emit DWARF/debug info.